packages/ui/src/molecules/QuantitySelector/QuantitySelector.tsx

```diff
@@ -0,0 +1,82 @@
+import type { InputHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+import type { InputProps } from '../../atoms/Input'
+import Input from '../../atoms/Input'
+import type { IconButtonProps } from '../IconButton'
+import IconButton from '../IconButton'
+
+export interface QuantitySelectorProps
+  extends InputHTMLAttributes<HTMLDivElement> {
+  /**
+   * Sets the current value that should be displayed on the input at the center of the quantity selector.
+   */
+  quantity: number | string
```

Comment:
Just out of curiosity, in which case we might have a string instead of number here? 👀

Reply:
I think the user might want to add sufixes to be displayed in the input, like "1 kg" or "2 meters". So I found it useful to allow a string to be accepted (naturally, the user would have to handle the custom behavior of the increment/decrement with their own functions)

Reply:
Oh okay! I was about to ask about the handles. Great!